UPGRADING.md

```diff
@@ -1,196 +1,13 @@
-but can easily be refreshed in the `Sigma Repos` tab using the `Refresh` menu item for each repository.
-
-## Removal of deprecated Inputs
-
-The following inputs are no longer available:
-- AWS Logs (deprecated)
-- AWS Flow Logs (deprecated)
-
-The inputs were marked as deprecated since Graylog version `3.2.0`.
-If you still run any of those inputs, please configure the alternative "Kinesis/CloudWatch" input instead ahead of upgrading.
-
-## Java API Changes
-The following Java Code API changes have been made.
-
-| File/method                                  | Description                                                                                                 |
-|----------------------------------------------|-------------------------------------------------------------------------------------------------------------|
-| `IndexSetValidator#validateRefreshInterval`  | The method argument have changed from `IndexSetConfig` to `Duration`                                        |
-| `IndexSetValidator#validateRetentionPeriod`  | The method argument have changed from `IndexSetConfig` to `RotationStrategyConfig, RetentionStrategyConfig` |
-| `ElasticsearchConfiguration#getIndexPrefix`  | The method name has changed to `getDefaultIndexPrefix`                                                      |
-| `ElasticsearchConfiguration#getTemplateName` | The method name has changed to `getDefaultIndexTemplateName`                                                |
-| `AuthServiceBackendConfig#externalHTTPHosts` | This method was added to the interface                                                                      |
-
-
-All previously deprecated index set configuration properties in `org.graylog2.configuration.ElasticsearchConfiguration`
-have been un-deprecated, as Graylog intends to maintain them going forward. 
-
-## REST API Endpoint Changes
-
-| Endpoint                                                                                                   | Description                                                                                                                                                                                                                                                                                                                                                                                                |
-|------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Not a big deal but we left the other h2 headers, do we want to leave these as well along with their formatting?